backend/src/sync.js

```diff
@@ -85,6 +86,18 @@ async function saveBlocks(blocksInfo) {
                   };
                 })
               );
+              models.Account.bulkCreate(
+                blockInfo.transactions.map(tx => {
+                  return {
+                    accountId: tx.receiver_id,
+                    transactionHash: tx.hash,
+                    timestamp
+                  };
+                })
+              ).catch(function(err) {
+                console.error(err);
+              });
+              return;
```

Comment:
This should return a promise. Thus, combine the two above `bulkCreate` calls with `Promise.all([...])` and return that.

Reply:
changed